crates/ui/src/windows/tilesets.rs

```diff
@@ -0,0 +1,764 @@
+            Overlay::Direction(_) => egui::vec2(-1., 0.),
+            _ => egui::Vec2::ZERO,
+        },
+        egui::Align2::CENTER_CENTER,
+        text,
+        egui::FontId {
+            size: match overlay {
+                Overlay::Passage(Passage::X) => 10.,
+                Overlay::Passage(Passage::O) => 14.,
+                Overlay::Passage(Passage::Square) => 18.,
+                Overlay::Direction(_) => 16.,
+                Overlay::Character(_) => 10.,
+                Overlay::Approx => 24.,
+                Overlay::Diamond => 24.,
+                Overlay::Dot => 32.,
+            },
+            family: egui::FontFamily::Name("Iosevka Term".into()),
+        },
+        egui::Color32::BLACK.gamma_multiply(if hovered { 0.8 } else { 0.3 }),
+    );
+
+    ui.painter().text(
+        pos + match overlay {
+            Overlay::Passage(Passage::Square) => egui::vec2(0., 1.),
+            Overlay::Direction(_) => egui::vec2(-1., 0.),
+            Overlay::Dot => egui::vec2(0., 1.),
+            _ => egui::Vec2::ZERO,
+        },
+        egui::Align2::CENTER_CENTER,
+        text,
+        egui::FontId {
+            size: match overlay {
+                Overlay::Passage(Passage::X) => 19.,
+                Overlay::Passage(Passage::O) => 27.,
+                Overlay::Passage(Passage::Square) => 38.,
+                Overlay::Direction(_) => 19.,
+                Overlay::Character(_) => 19.,
+                Overlay::Approx => 27.,
+                Overlay::Diamond => 27.,
+                Overlay::Dot => 38.,
+            },
+            family: egui::FontFamily::Name("Iosevka Term".into()),
+        },
+        egui::Color32::BLACK.gamma_multiply(if hovered { 0.8 } else { 0.3 }),
+    );
+
+    ui.painter().text(
+        pos + match overlay {
+            Overlay::Direction(_) => egui::vec2(-1., 0.),
+            _ => egui::Vec2::ZERO,
+        },
+        egui::Align2::CENTER_CENTER,
+        text,
+        egui::FontId {
+            size: match overlay {
+                Overlay::Passage(Passage::X) => 16.,
+                Overlay::Passage(Passage::O) => 24.,
+                Overlay::Passage(Passage::Square) => 32.,
+                Overlay::Direction(_) => 16.,
+                Overlay::Character(_) => 16.,
+                Overlay::Approx => 24.,
+                Overlay::Diamond => 24.,
+                Overlay::Dot => 32.,
+            },
+            family: egui::FontFamily::Name("Iosevka Term".into()),
+        },
+        egui::Color32::WHITE.gamma_multiply(if hovered { 0.9 } else { 0.5 }),
+    );
+}
+
+/// Database - Tilesets management window.
+pub struct Window {
+    selected_tileset_name: Option<String>,
+    property: Property,
+
+    previous_tileset: Option<usize>,
+
+    autotile_modals: [AutotileModal; 7],
+    tileset_modal: TilesetModal,
+    panorama_modal: PanoramaModal,
+    fog_modal: FogModal,
+    battleback_modal: LabelModal,
+
+    tilepicker: Option<Tilepicker>,
```

Comment:
The tilepicker doesn't have a big enough minimum size, causing a small portion can be visible if the window isn't big enough
![image](https://github.com/user-attachments/assets/14191b40-afad-4521-a7ea-5e00d315b3ec)